### DISCUSSION TITLE
Question about "Restore last states on startup"

### DISCUSSION THREAD

**Discussion by WesWesthaver** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#635] [Q&A]
I recently had to kill the power on the circuit that feeds my Lay-Z-Spa. When I powered up the circuit again, the spa switched back to displaying the temperature in Celsius. I have the "Restore last states on startup" option checked but it didn't remember my preference for Fahrenheit. Is this a bug?

Thanks,
-Wes

**Comment by visualapproach** [comment]
Hi, maybe, or the power went out just as the ESP wrote to flash and it destroyed the info. Or something else. It should restore states on startup.
